docs/tutorial.md

```diff
@@ -228,7 +245,7 @@ This indicates that all tasks within the workflow is complete.
 
 ### Accessing data
 
-In your directory, CytoSnake produces a `results` folder, which will
+In your directory, `CytoSnake` produces a `results/` folder, which will
 contain all the outputs generated from the workflow. To list those
```

Comment:
```suggestion
contain all the outputs generated from the workflow. 
To list those
```
Not sure if the formatting on my end is wrong, but it looks like this sentence should be on another line.